konnect-runtime-setup.sh

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 
 KONNECT_RUNTIME_PORT=8000
-KONNECT_API_URL=
+KONNECT_API_HOST=
+KONNECT_GEO=
```

Comment:
https://github.com/Kong/kbilling/pull/58#discussion_r933591102

Sorry for the late comment. we need to rename `geo` to 'region` since this script is customer facing.